keps/sig-instrumentation/2305-metrics-cardinality-enforcement/kep.yaml

```diff
@@ -14,6 +15,9 @@ reviewers:
 approvers:
   - sig-instrumentation
```

Comment:
Approvers need to be individuals, you can probably set me as an approver.

Reply:
Got you. Thanks